thompickett/rails_engine
# Translate provided cvs files into pg tables

* Create following Models and Migrations
   * Customer
   * Merchant
   * Item
   * Invoice
   * InvoiceItem
   * Transaction 
* Set up rspec for testing  
* Create model_spec for all of above models   
testing the relationships to each other using  
the should-matcher gems, belong_to and have_many  
methods. 
* Add the gems to the gemfile
* Under test
	* shoulda-matchers, 3.1
	* simplecov
        * launchy
        * capybara
        * rspec-rails
* Under test, production
        * pry-rails
        * remove byebug
* Create readme with section showing user how to  
set up the app and migrate in the cvs files
